README.md

```diff
@@ -51,7 +51,7 @@ Following matrix shows the platform specific support currently available.
 | [Prism.Unity][5] <sup>(*)</sup>  | [![25]][5] |  X  |  X  |     |
 | [Prism.Unity.Forms][10]| [![30]][10]|     |     |  X  |
 | [Prism.Mef][6]  <sup>(**)</sup> | [![26]][6] |  X  | - | - |
-| [Prism.Autofac][7]     | [![27]][7] |  X  |  X  |     |
+| [Prism.Autofac][7]     | [![27]][7] |  X  |  X  |  X  |
```

Comment:
We will need a separate Prism.Autofac.Forms package and the PS build script updated to build the packages.